motoman_driver/src/industrial_robot_client/joint_trajectory_action.cpp

```diff
@@ -532,18 +554,19 @@ void JointTrajectoryAction::controllerStateCB(
     return;
   }
 
-  if (!industrial_utils::isSimilar(all_joint_names_, msg->joint_names))
+  // joint torso_joint_b2 is not in msg->joint_names, so this code is commented out.
```

Comment:
yes, it would be much better. I will firstly vertify this bug (publisher for dual-arm robots), after a couple of days.